src/Plugin/Knn/Handler.php

```diff
@@ -178,6 +156,33 @@ private static function knnSqlQuery(Client $manticoreClient, Payload $payload, s
 			}
 		}
 
+
 		return $result;
 	}
+
+	/**
```

Comment:
Missing doc info about this method cuz from the name it's actually not very clear what it does for me when you are no in the context